Start Agent init....
Entered InverseKinematics Env
Entered Moveo Env
[ INFO] [1627364290.523837178]: Loading robot model 'moveo_description'...
[ INFO] [1627364290.524614056]: No root/virtual joint specified in SRDF. Assuming fixed joint
[31m[ERROR] [1627364291.799693, 2742.219000]: NOT Initialising Simulation Physics Parameters
[33m[WARN] [1627364291.823885, 2742.243000]: Start Init ControllersConnection
[33m[WARN] [1627364291.826098, 2742.243000]: END Init ControllersConnection
/home/nils/anaconda3/envs/moveo_training/lib/python3.9/site-packages/gym/logger.py:30: UserWarning: [33mWARN: Box bound precision lowered by casting to float32
  warnings.warn(colorize('%s: %s'%('WARN', msg % args), 'yellow'))
2021-07-27 07:38:12.096345: I tensorflow/compiler/jit/xla_cpu_device.cc:41] Not creating XLA devices, tf_xla_enable_xla_devices not set
2021-07-27 07:38:12.097028: I tensorflow/core/platform/cpu_feature_guard.cc:142] This TensorFlow binary is optimized with oneAPI Deep Neural Network Library (oneDNN) to use the following CPU instructions in performance-critical operations:  SSE4.1 SSE4.2 AVX AVX2 FMA
To enable them in other operations, rebuild TensorFlow with the appropriate compiler flags.
2021-07-27 07:38:12.098768: I tensorflow/core/common_runtime/process_util.cc:146] Creating new thread pool with default inter op setting: 2. Tune using inter_op_parallelism_threads for best performance.
###### Setting Group Moveit with 30 seconds wait...
[ INFO] [1627364291.755169434, 2742.180000000]: Ready to take commands for planning group arm.
<class 'moveo_commander.MoveoCommander'>
launch the init function of the Parent Class robot_gazebo_env_goal.RobotGazeboEnv.....
Entered Gazebo Env
False
launch the init function of the Parent Class robot_gazebo_env_goal.RobotGazeboEnv.....END
Entered Moveo Env END
START Actor init
END Actor init....
NUMBER OF WORKERS=CPUs available for this training=1
END Agent init....
Entered InverseKinematics Env
Entered Moveo Env
###### Setting Group Moveit with 30 seconds wait...
[ INFO] [1627364292.389445554, 2742.772000000]: Ready to take commands for planning group arm.
<class 'moveo_commander.MoveoCommander'>
launch the init function of the Parent Class robot_gazebo_env_goal.RobotGazeboEnv.....
Entered Gazebo Env
False
launch the init function of the Parent Class robot_gazebo_env_goal.RobotGazeboEnv.....END
Entered Moveo Env END
Start Worker Agent init....
START Actor init
END Actor init....
END Worker Agent init....
####################### START Train
####################### START AC3
EP0 EpisodeReward=0
Entered reset
0.0
0.0
0.0
0.0
0.0
Entered step
Unpause sim
Set action
Action:
[-0.15659083  1.11872856 -0.52079418  0.91150763  0.14717067]
-0.15659082720634454
1.1187285618975904
-0.5207941790687335
0.9115076291048712
0.1471706729540857
[31m[ERROR] [1627364292.417044, 2742.791000]: NOT Initialising Simulation Physics Parameters
[33m[WARN] [1627364292.445012, 2742.816000]: Start Init ControllersConnection
[33m[WARN] [1627364292.446332, 2742.817000]: END Init ControllersConnection
2021-07-27 07:38:13.309284: I tensorflow/compiler/mlir/mlir_graph_optimization_pass.cc:116] None of the MLIR optimization passes are enabled (registered 2)
2021-07-27 07:38:13.327136: I tensorflow/core/platform/profile_utils/cpu_utils.cc:112] CPU Frequency: 3192790000 Hz
[ INFO] [1627364305.806173992, 2755.715000000]: ABORTED: Solution found but controller failed during execution
Get Obs
Is done
>>>>>>>>>>>>>>>>done_fail=False,done_sucess=True
Original Shape==(5,)
Morfed Shape==(1, 5)
EP0 EpisodeReward=100
####################### START AC3
EP1 EpisodeReward=0
Entered reset
0.0
0.0
0.0
0.0
0.0
[ INFO] [1627364317.646531856, 2767.344000000]: ABORTED: Solution found but controller failed during execution
Entered step
Unpause sim
Set action
Action:
[-0.45827994 -0.03777554 -0.71468035  0.05184965 -0.58341594]
-0.45827993955425933
-0.03777554330772159
-0.7146803529758475
0.05184964972457009
-0.583415939415796
[ INFO] [1627364324.967879640, 2774.530000000]: ABORTED: Solution found but controller failed during execution
Get Obs
Is done
>>>>>>>>>>>>>>>>done_fail=False,done_sucess=True
Original Shape==(5,)
Morfed Shape==(1, 5)
EP1 EpisodeReward=100
####################### START AC3
EP2 EpisodeReward=0
Entered reset
0.0
0.0
0.0
0.0
0.0
Exception in thread Thread-23:
Traceback (most recent call last):
  File "/home/nils/anaconda3/envs/moveo_training/lib/python3.9/threading.py", line 954, in _bootstrap_inner
    self.run()
  File "/home/nils/Documents/Studium/Bachelorarbeit/moveo_RL/src/moveo_training/scripts/A3C_moveo_ik.py", line 256, in run
    self.train()
  File "/home/nils/Documents/Studium/Bachelorarbeit/moveo_RL/src/moveo_training/scripts/A3C_moveo_ik.py", line 201, in train
    state = self.env.reset()
  File "/home/nils/anaconda3/envs/moveo_training/lib/python3.9/site-packages/gym/wrappers/time_limit.py", line 25, in reset
    return self.env.reset(**kwargs)
  File "/home/nils/Documents/Studium/Bachelorarbeit/moveo_RL/src/openai_ros/openai_ros/src/openai_ros/robot_gazebo_env_goal.py", line 63, in reset
    self._reset_sim()
  File "/home/nils/Documents/Studium/Bachelorarbeit/moveo_RL/src/openai_ros/openai_ros/src/openai_ros/robot_gazebo_env_goal.py", line 121, in _reset_sim
    self.gazebo.resetWorld()
  File "/home/nils/Documents/Studium/Bachelorarbeit/moveo_RL/src/openai_ros/openai_ros/src/openai_ros/gazebo_connection.py", line 106, in resetWorld
    rospy.wait_for_service('/gazebo/reset_world')
  File "/opt/ros/noetic/lib/python3/dist-packages/rospy/impl/tcpros_service.py", line 166, in wait_for_service
    raise ROSInterruptException("rospy shutdown")
rospy.exceptions.ROSInterruptException: rospy shutdown
[ INFO] [1627364332.979918885, 2782.161000000]: ABORTED: Solution found but controller failed during execution